core/src/edu/asu/c3simulator/screens/BusinessCreationGuide.java

```diff
@@ -0,0 +1,218 @@
+		String screenName;
+		
+		/**
+		 * @param targetedScreen
+		 *            The screen which the button will show.
+		 * @param screenName
+		 *            The name of the screen.
+		 */
+		public navigationButtonListener(BusinessCreationGuideScreens targetedScreen,
+				String screenName)
+		{
+			this.targetedScreen = targetedScreen;
+			this.screenName = screenName;
+		}
+		
+		@Override
+		public void clicked(InputEvent event, float x, float y)
+		{
+			System.out.println(screenName);
+			showScreen(targetedScreen);
+		}
+	}
+	
+	private class progressButtonListener extends ClickListener
+	{
+		@Override
+		public void clicked(InputEvent event, float x, float y)
+		{
+			if (currentScreen == industryScreen)
+			{
+				System.out.println("Transitioning to Direction Screen");
+				showScreen(directionScreen);
+			}
+			else if (currentScreen == directionScreen)
+			{
+				System.out.println("Transitioning to funding Screen");
+				showScreen(fundingScreen);
+			}
+			else if (currentScreen == fundingScreen)
+			{
+				System.out.println("Transitioning to tasks Screen");
+				showScreen(tasksScreen);
+			}
+			else if (currentScreen == tasksScreen)
+			{
+				System.out.println("Transitioning to Home Screen");
+				// TODO: Transition to main hub
+				// TODO: pass user choices to data layer
+			}
+		}
+	}
+	
+	@SuppressWarnings("unused")
+	private Game game;
+	private Stage stage;
+	private Skin skin;
+	private BusinessCreationGuideScreens industryScreen, directionScreen, fundingScreen,
+			tasksScreen, currentScreen;
+	TextButton progressButton;
+	
+	public BusinessCreationGuide(Game game)
+	{
+		this.game = game;
+		this.stage = new Stage();
+		this.skin = new Skin(Gdx.files.internal("skins/default/uiskin.json"));
+		
+		industryScreen = new IndustryScreen(stage); // IndustryScreen();
+		directionScreen = new BusinessDirectonScreen(stage);
+		fundingScreen = new FundingScreen(stage); // FundingScreen();
+		tasksScreen = new TasksScreen(stage); // TasksScreen();
+		
+		TextButton industryButton = new TextButton("Industry", skin);
+		TextButton directonButton = new TextButton("Directon", skin);
+		TextButton fundingButton = new TextButton("Funding", skin);
+		TextButton tasksButton = new TextButton("Tasks", skin);
+		
+		industryButton.addListener(new navigationButtonListener(industryScreen,
+				"Industry"));
+		directonButton.addListener(new navigationButtonListener(directionScreen,
+				"Directon"));
+		fundingButton.addListener(new navigationButtonListener(fundingScreen, "Funding"));
+		tasksButton.addListener(new navigationButtonListener(tasksScreen, "Tasks"));
+		
+		Table navigationButtons = new Table();
+		navigationButtons.add(industryButton).fillX().padTop(10);
+		navigationButtons.row();
+		navigationButtons.add(directonButton).fillX().padTop(10);
+		navigationButtons.row();
+		navigationButtons.add(fundingButton).fillX().padTop(10);
+		navigationButtons.row();
+		navigationButtons.add(tasksButton).fillX().padTop(10);
```

Comment:
I did consider that but Table gives me more freedom in customizing each and every cell of the table while it is being added like fitting the widget to the cell's size. I found VerticalGroup not to be as flexible.